packages/orbit-components/src/Tile/Tile.stories.tsx

```diff
@@ -14,284 +14,209 @@ import Tile from ".";
-      type: "select",
+  parameters: {
+    controls: {
+      exclude: ["onClick", "external", "noPadding", "initialExpanded", "noHeaderIcon"],
     },
   },
-};
 
-export const Group = () => {
-  return (
-    <TileGroup as="ul">
-      <Tile as="li" href="#">
-        List Item
-      </Tile>
-      <Tile as="li" href="#">
-        List Item
-      </Tile>
-      <Tile as="li" href="#">
-        List Item
-      </Tile>
-      <Tile as="li" href="#">
-        List Item
-      </Tile>
-    </TileGroup>
-  );
+  args: {
+    showMore: false,
+    icon: "GenderMan",
+    expandable: true,
+    children: "This is example of expanded content",
+  },
 };
 
-export const Playground = ({
-  href,
-  title,
-  external,
-  icon,
-  description,
-  header,
-  expandable,
-  initialExpanded,
-  noPadding,
-  dataTest,
-  children,
-  htmlTitle,
-  as,
-  expanded,
-}) => {
-  const Icon = getIcon(icon);
+export const Playground: Story = {
+  render: ({ icon, children, ...args }) => {
+    const Icon = getIcon(icon);
 
-  return (
-    <Tile
-      href={href}
-      as={as === "" ? undefined : "div"}
-      onClick={action("clicked")}
-      icon={Icon && <Icon />}
-      expanded={expanded}
-      title={title}
-      description={description}
-      header={header}
-      external={external}
-      expandable={expandable}
-      initialExpanded={initialExpanded}
-      noPadding={noPadding}
-      dataTest={dataTest}
-      htmlTitle={htmlTitle}
-    >
-      {children}
-    </Tile>
-  );
-};
+    return (
+      <Tile {...args} icon={Icon && <Icon />}>
+        {children}
+      </Tile>
+    );
+  },
 
-Playground.story = {
   parameters: {
-    info: "This is the default configuration of this component.",
+    info: "This is the playground configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      exclude: ["onClick"],
+    },
   },
-};
-
-Playground.args = {
-  href: "https://www.kiwi.com/",
-  title: "Tile with title",
-  external: false,
-  icon: "Airplane",
-  description:
-    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco.",
-  header: "",
-  expandable: false,
-  initialExpanded: false,
-  noPadding: false,
-  dataTest: "test",
-  children: "",
-  htmlTitle: "Title for more info",
-  as: "",
-  expanded: false,
-};
 
-Playground.argTypes = {
-  icon: {
-    options: Object.keys(Icons),
-    control: {
-      type: "select",
-    },
+  args: {
+    href: "https://www.kiwi.com/",
+    external: false,
+    id: "ID",
+    icon: "Airplane",
+    title: "Tile with title",
+    expanded: false,
+    description:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco.",
+    header: "",
+    children: "",
+    noPadding: false,
```

Comment:
in the Playground, `noPadding` is applied only for the "children" element, but it could also be applicable for "parent". I couldn't set it for the parent, but it should be working (I added `noPadding` for the first default story):

<img width="1088" alt="image" src="https://github.com/user-attachments/assets/63dc14d9-68af-4a42-9730-e993773604d0">

Can you take a look at it and include it to the Playground as well? 🙏

Reply:
No, it cannot be included. In your example you don't have header for the tile (= title, description, icon, header) so `noPadding` is applied to the same element as in the Playground which is `TileContent` component.